server/fishtest/userdb.py

```diff
@@ -137,12 +137,23 @@ def save_user(self, user):
         self.last_blocked_time = 0
         self.clear_cache()
 
-    def remove_user(self, user):
+    def remove_user(self, rejector, user):
+
         result = self.users.delete_one({"_id": user["_id"]})
         if result.deleted_count > 0:
             # User successfully deleted
             self.last_pending_time = 0
             self.clear_cache()
+
+            # Keep in logs the rejected users in case it was by mistake
+            # so we can email the user to register again
+            # and documents who rejected the user since we don't want to report it in the event log
+            if rejector:
+                current_time = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+                print(
+                    f"{current_time}: user: {user['username']} with the email: {user['email']} got rejected by: {rejector}",
+                    flush=True,
+                )
```

Comment:
The server already adds a timestamp to the log entry. So there is no need to do it here.

Reply:
Thanks, I'll remove that

Reply:
Perhaps also 
```
user: {user['username']} with email: {user['email']} was rejected by: {rejector}
```
(nitpicking)